brands/shop/frozen_food.json

```diff
@@ -1,4 +1,21 @@
 {
+  "shop/frozen_food|Dream Dinners": {
+    "countryCodes": ["us"],
+    "matchTags": [
+      "amenity/fast_food",
+      "amenity/restaurant",
+      "shop/food",
+      "shop/grocery"
+    ],
+    "tags": {
+      "brand": "Dream Dinners",
+      "brand:wikidata": "Q5306355",
+      "brand:wikipedia": "en:Dream Dinners",
+      "name": "Dream Dinners",
+      "opening_hours": "\"by appointment\"",
```

Comment:
Mappers who know the hours during which appointments are available can prepend the usual `opening_hours` syntax to this comment.